### PR TITLE
Resolve warnings

### PR DESCRIPTION
**Pull request by dkp1903**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1624,14 +1624,6 @@
...
       }
     },
-    "@material-ui/icons": {
-      "version": "4.9.1",
-      "resolved": "https://registry.npmjs.org/@material-ui/icons/-/icons-4.9.1.tgz",
-      "integrity": "sha512-GBitL3oBWO0hzBhvA9KxqcowRUsA0qzwKkURyC8nppnC3fw54KPKZ+d4V1Eeg/UnDRSzDaI9nGCdel/eh9AQMg==",
-      "requires": {
-        "@babel/runtime": "^7.4.4"
-      }
-    },
```
Comment (pranavpandey1998official) [inline package-lock.json:1634]: why you delete this bro

File: .gitignore [hunk 2]
```diff
@@ -8,6 +8,7 @@
 # testing
 /coverage
 
+/.vscode
```
Comment (pranavpandey1998official) [inline .gitignore:11]: why

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+    "deno.enable": false
+}
```
Comment (pranavpandey1998official) [inline .vscode/settings.json:3]: why

**Comment by dkp1903** [comment]
Sorry bro, i had no idea package.json me koi change hua. Bcz I had pulled
your latest changes, and uske baad npm.i kuchh nahi kiya.

Vo vs code wala happened bcz mere VS code me Deno extension enabled tha, to
usko disable karne ke liye aisa hua tha. But then, i just removed the
extension, and added the vscode folder to ignore

**Pull request merged**